app/client/cypress/setup-test.sh

```diff
@@ -24,6 +24,5 @@ echo "Sleeping for 5 seconds to let the server start"
 sleep 5
 
 DEBUG=cypress:* $(npm bin)/cypress version
-echo $CYPRESS_URL
-sed -i -e 's|api_url:.*$|api_url: "$CYPRESS_URL"|g' /github/home/.cache/Cypress/4.1.0/Cypress/resources/app/packages/server/config/app.yml
+sed -i -e "s|api_url:.*$|api_url: $CYPRESS_URL|g" /github/home/.cache/Cypress/4.1.0/Cypress/resources/app/packages/server/config/app.yml
```

Comment:
I'd recommend the following here. URLs in YAML without double quotes are known to screw things.

```suggestion
sed -i -e 's|api_url:.*$|api_url: "'"$CYPRESS_URL"'"|g' /github/home/.cache/Cypress/4.1.0/Cypress/resources/app/packages/server/config/app.yml
```